test/tests/unit/wire_protocol_test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bson = require('bson');
+const wireProtocol2_6 = require('../../../lib/wireprotocol/2_6_support.js');
+const wireProtocol3_2 = require('../../../lib/wireprotocol/3_2_support.js');
+
+const MockPool = {};
+
+MockPool.write = function(cmd, options, callback) {
+  callback(null, cmd);
+};
+
+describe('WireProtocol', function() {
+  it('2.6 should only set bypassDocumentValidation to true if explicitly set by user to true', function(done) {
+    const options = { bypassDocumentValidation: true };
+    wireProtocol2_6.prototype.insert(MockPool, '', '', bson, 'ops', options, function(err, data) {
```

Comment:
similarly I would recommend creating an instance of the `WireProtocol`: `const wireProtocol = new WireProtocol2_6();` and using `wireProtocol.insert`